static/js/list_rendering.js

```diff
@@ -120,6 +120,8 @@ var list_render = (function () {
                 return this;
             },
 
+            // clear the HTML and reset the offset. This is good for DOM memory
+            // cleanup.
```

Comment:
My thought was that this comment was probably unnecessary -- it's clear from looking at the code that this is what it does.